src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import duke.DukeException;
+
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+import static java.lang.String.format;
+
+/**
+ * Ui class for user interaction and to manage interface.
+ *
+ */
+public class Ui {
+    private String line;
+    Scanner sc;
```

Comment:
Is there any reason that `sc` is not a private attribute? Do you need to access it anywhere outside the `Ui` class?

Reply:
Thank you for your comments! It helped me improved my code quality.